StrictEmit/ILGeneratorExtensions.Objects.cs

```diff
@@ -30,7 +30,7 @@ public static partial class ILGeneratorExtensions
         /// Creates a new object or a new instance of a value type, pushing an object reference
         /// (type <strong>O</strong>) onto the evaluation stack.
         /// </summary>
-        /// <typeparam name="T">The type.</typeparam>
+        /// <typeparam name="T">The type of object to emit.</typeparam>
```

Comment:
We're not emitting an object, we're emitting an opcode. Perhaps "to create an instance of"? Same for the others.